tests/jeos/firstrun.pm

```diff
@@ -72,7 +72,12 @@ sub run {
     mouse_hide;
     # kiwi-templates-JeOS images (sle, opensuse x86_64 only) are build w/o translations
     # jeos-firstboot >= 0.0+git20200827.e920a15 locale warning dialog has been removed
-    if ((is_sle('15+') && is_sle('<15-sp3')) || (is_leap('<15.3') && is_x86_64)) {
+    if (is_tumbleweed || is_leap('15.4+')) {
```

Comment:
It'll reach SLE before Leap, but for now it should be just `is_tumbleweed` or `unless (is_sle('<=15-sp4') || is_leap('<=15.4'))`